### ISSUE TITLE
[BITV] 9.1.1.1c/2.2 - Modal dialogs "Online status" and "Edit widgets" / "Weather" menu - Decorative icons were not marked as such with aria-hidden="true". E.g. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36909]
**Blocked. First new NcVue version should be released and all bumps to master, stable25, stable26 have to be done.**

- [ ] Fix
- [ ] Backport

i.e. ```<div data-v-7d1a0f30="" role="img" class="icon-weather-status"></div>```

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3a2875951a-v2-n2

</details>

**Comment by JuliaKirschenheuter** [comment]
https://www.sarasoueidan.com/blog/accessible-icon-buttons/#icon-sitting-next-to-text

**Comment by JuliaKirschenheuter** [comment]
Fix:
- [ ] https://github.com/nextcloud/server/pull/37335
- [ ] https://github.com/nextcloud/nextcloud-vue/pull/3931
Backport:
